popup.js

```diff
@@ -72,6 +72,197 @@ const containersworks = [
   },
 ];
 
+const Workcards = document.getElementById('containers');
+
+Workcards.innerHTML = `
```

Comment:
As requested by the previous reviewers, this section should be reading data 📊 from the `containersworks`. array of objects, for now, your project section is hardcoded. You should follow the same model you have for the modal below, I suggest a loop or a map tho. 🤓